package.json

```
@@ -76,16 +76,16 @@
     "vkbeautify": "^0.99.3"
   },
   "peerDependencies": {
-    "pixi.js": "^5.0.3",
+    "pixi.js": "^6.3.0",
```

This is kind of surprising to me that this is the only pixi change. I expected also to see pixi change in our dev dependencies, but we already have pixi@6 listed there... Maybe we have already been using pixi@6 for development and testing for while?

---

Like I don't see any pixi changes in the lockfile, except for `peerDependencies`.

---

If it really is the case that v5 works _and_ v6 works, then maybe we can do something similar to react/react-dom?

```suggestion
    "pixi.js": "^5.0.3 || ^6.3.0",
```

---

> Maybe we have already been using pixi@6 for development and testing for while?

I think that's correct! It seem higlass was already using pixi v6 for dev, it just never got updated as a peer dep? Just changed so that pixi peer can be v5 or v6.

---

But, in this case, I believe the issue ([gosling.js/#977](https://github.com/gosling-lang/gosling.js/pull/977)) with the `@pixi/polyfill` would remain unaddressed?

---

Yes, I would assume that esm.sh would pick the latest pixi version from the available peer dependencies.